src/commands/starter/starter.command.ts

```diff
@@ -0,0 +1,121 @@
+                    \n [Shaun Spalding](https://www.youtube.com/user/999Greyfox) \
+                    \n [HeartBeast](https://www.youtube.com/user/uheartbeast) \
+                    \n [Freindly Cosmonaut](https://www.youtube.com/channel/UCKCKHxkH8zqV9ltWZw0JFig) \
+                    \n [Very useful bite sized videos on specific topics](https://www.youtube.com/channel/UCcYKLm0EwyWkfTA6sMn5W7g) \
+                    \n [Official YYG Youtube](https://www.youtube.com/user/yoyogamesltd)'
+                  },
+
+                  {
+                    name: '**Advanced GML Topics:**',
+                    value: '[Git based source control with GMS2](https://www.youtube.com/watch?v=6zj86KN8Vco) \
+                    \n [Debug Mode Basics](https://www.youtube.com/watch?v=iJH_uTq9iOQ)'
+                  },
+
+                  {
+                    name: '**Basic Art Skill Articles:**',
+                    value: '[Guide to Being A Respectful Critic](http://tiny.cc/critique) \
+                    \n [Basic Guide to Pixelart](http://pixeljoint.com/forum/forum_posts.asp?TID=11299)'
+                  },
+
+                  {
+                    name: '**Art Video Recources and Channels:**',
+                    value: '[MortMort](https://www.youtube.com/user/atMNRArt) \
+                    \n [Guide on Choosing the Right Canvas Size](https://www.youtube.com/watch?v=AXb-VBZTKDA)'
+                  },
+
+                  {
+                    name: '**Pixelart Refrence Sites:**',
+                    value: '[Miniboss](http://blog.studiominiboss.com/pixelart) \
+                    \n [The Spriters Recource](https://www.spriters-resource.com/) \n'
+                  },
+
+                  {
+                    name: '**General Pixelart Tips:**',
+                    value: '-Start at a low resolution \
+                    \n -Use a pallete \
+                    \n -Use games you like as refrence \
+                    \n [Find Palletes Here](https://lospec.com/palette-list)'
+                  },
+
+                  {
+                    name: '**Music Theory and Learning Sites/Channels:**',
+                    value: '[MusicTheory.net](https://www.musictheory.net/lessons) \
+                    \n [12tone Building Blocks](https://www.youtube.com/playlist?list=PLMvVESrbjBWplAcg3pG0TesncGT7qvO06)'
+                  },
+
+                  {
+                    name: '**Other Useful Topics and Recources:**',
+                    value: '["Your First Game Will (and Should) Suck" By: PixelatedPope](https://goo.gl/sgz7d2) \
+                    \n [Your Game Idea is Too Big](http://yourgameideaistoobig.com) \
+                    \n [Rubber Duck Debugging](https://rubberduckdebugging.com/)'
+                  }
+                ],
+                color: defaultEmbedColor
+              });
+            let reciver = client.fetchUser(whoTag).then(user => {
```

Comment:
We don't need this.
`whoTag` is a "User" type, not a string. So we can just directly do `whoTag.send(kitEmbed)` without this promisification. I'd run that locally to make sure, but it looks good to me.